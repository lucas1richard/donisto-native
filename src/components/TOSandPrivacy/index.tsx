import React from 'react';
import { View } from 'react-native';
import Txt from 'components/Txt';
import Touchable from 'components/Touchable';

function TOSandPrivacy() {
  return (
    <View style={{ alignItems: 'center' }}>
      <Txt color="light">
        By Proceeding, I agree to the
      </Txt>
      <View style={{ flexDirection: 'row', justifyContent: 'center' }}>
        <Touchable>
          <View>
            <Txt color="light" style={{ fontWeight: '800' }}>
              Terms of Use
            </Txt>
          </View>
        </Touchable>
        <Txt color="light">
          &nbsp;and&nbsp;
        </Txt>
        <Touchable>
          <View>
            <Txt color="light" style={{ fontWeight: '800' }}>
              Privacy Policy
            </Txt>
          </View>
        </Touchable>
      </View>
    </View>
  );
}

export default TOSandPrivacy;
