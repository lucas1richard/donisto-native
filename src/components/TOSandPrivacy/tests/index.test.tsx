import * as React from 'react';
import { shallow } from 'enzyme';
import TOSandPrivacy from '..';

describe('<TOSandPrivacy />', () => {
  const renderComponent = () => shallow(<TOSandPrivacy />);
  it('renders', () => {
    expect(renderComponent()).toMatchSnapshot();
  });
});