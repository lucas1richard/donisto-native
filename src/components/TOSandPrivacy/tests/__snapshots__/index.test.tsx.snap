// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TOSandPrivacy /> renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TOSandPrivacy />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <Txt
          color="light"
        >
          By Proceeding, I agree to the
        </Txt>,
        <Component
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "center",
            }
          }
        >
          <TouchableOpacity
            activeOpacity={0.2}
          >
            <Component>
              <Txt
                color="light"
                style={
                  Object {
                    "fontWeight": "800",
                  }
                }
              >
                Terms of Use
              </Txt>
            </Component>
          </TouchableOpacity>
          <Txt
            color="light"
          >
             and 
          </Txt>
          <TouchableOpacity
            activeOpacity={0.2}
          >
            <Component>
              <Txt
                color="light"
                style={
                  Object {
                    "fontWeight": "800",
                  }
                }
              >
                Privacy Policy
              </Txt>
            </Component>
          </TouchableOpacity>
        </Component>,
      ],
      "style": Object {
        "alignItems": "center",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "By Proceeding, I agree to the",
          "color": "light",
        },
        "ref": null,
        "rendered": "By Proceeding, I agree to the",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <TouchableOpacity
              activeOpacity={0.2}
            >
              <Component>
                <Txt
                  color="light"
                  style={
                    Object {
                      "fontWeight": "800",
                    }
                  }
                >
                  Terms of Use
                </Txt>
              </Component>
            </TouchableOpacity>,
            <Txt
              color="light"
            >
               and 
            </Txt>,
            <TouchableOpacity
              activeOpacity={0.2}
            >
              <Component>
                <Txt
                  color="light"
                  style={
                    Object {
                      "fontWeight": "800",
                    }
                  }
                >
                  Privacy Policy
                </Txt>
              </Component>
            </TouchableOpacity>,
          ],
          "style": Object {
            "flexDirection": "row",
            "justifyContent": "center",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "activeOpacity": 0.2,
              "children": <Component>
                <Txt
                  color="light"
                  style={
                    Object {
                      "fontWeight": "800",
                    }
                  }
                >
                  Terms of Use
                </Txt>
              </Component>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <Txt
                  color="light"
                  style={
                    Object {
                      "fontWeight": "800",
                    }
                  }
                >
                  Terms of Use
                </Txt>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": "Terms of Use",
                  "color": "light",
                  "style": Object {
                    "fontWeight": "800",
                  },
                },
                "ref": null,
                "rendered": "Terms of Use",
                "type": [Function],
              },
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": " and ",
              "color": "light",
            },
            "ref": null,
            "rendered": " and ",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "activeOpacity": 0.2,
              "children": <Component>
                <Txt
                  color="light"
                  style={
                    Object {
                      "fontWeight": "800",
                    }
                  }
                >
                  Privacy Policy
                </Txt>
              </Component>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <Txt
                  color="light"
                  style={
                    Object {
                      "fontWeight": "800",
                    }
                  }
                >
                  Privacy Policy
                </Txt>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": "Privacy Policy",
                  "color": "light",
                  "style": Object {
                    "fontWeight": "800",
                  },
                },
                "ref": null,
                "rendered": "Privacy Policy",
                "type": [Function],
              },
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Txt
            color="light"
          >
            By Proceeding, I agree to the
          </Txt>,
          <Component
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "center",
              }
            }
          >
            <TouchableOpacity
              activeOpacity={0.2}
            >
              <Component>
                <Txt
                  color="light"
                  style={
                    Object {
                      "fontWeight": "800",
                    }
                  }
                >
                  Terms of Use
                </Txt>
              </Component>
            </TouchableOpacity>
            <Txt
              color="light"
            >
               and 
            </Txt>
            <TouchableOpacity
              activeOpacity={0.2}
            >
              <Component>
                <Txt
                  color="light"
                  style={
                    Object {
                      "fontWeight": "800",
                    }
                  }
                >
                  Privacy Policy
                </Txt>
              </Component>
            </TouchableOpacity>
          </Component>,
        ],
        "style": Object {
          "alignItems": "center",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "By Proceeding, I agree to the",
            "color": "light",
          },
          "ref": null,
          "rendered": "By Proceeding, I agree to the",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <TouchableOpacity
                activeOpacity={0.2}
              >
                <Component>
                  <Txt
                    color="light"
                    style={
                      Object {
                        "fontWeight": "800",
                      }
                    }
                  >
                    Terms of Use
                  </Txt>
                </Component>
              </TouchableOpacity>,
              <Txt
                color="light"
              >
                 and 
              </Txt>,
              <TouchableOpacity
                activeOpacity={0.2}
              >
                <Component>
                  <Txt
                    color="light"
                    style={
                      Object {
                        "fontWeight": "800",
                      }
                    }
                  >
                    Privacy Policy
                  </Txt>
                </Component>
              </TouchableOpacity>,
            ],
            "style": Object {
              "flexDirection": "row",
              "justifyContent": "center",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "activeOpacity": 0.2,
                "children": <Component>
                  <Txt
                    color="light"
                    style={
                      Object {
                        "fontWeight": "800",
                      }
                    }
                  >
                    Terms of Use
                  </Txt>
                </Component>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Txt
                    color="light"
                    style={
                      Object {
                        "fontWeight": "800",
                      }
                    }
                  >
                    Terms of Use
                  </Txt>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": "Terms of Use",
                    "color": "light",
                    "style": Object {
                      "fontWeight": "800",
                    },
                  },
                  "ref": null,
                  "rendered": "Terms of Use",
                  "type": [Function],
                },
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": " and ",
                "color": "light",
              },
              "ref": null,
              "rendered": " and ",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "activeOpacity": 0.2,
                "children": <Component>
                  <Txt
                    color="light"
                    style={
                      Object {
                        "fontWeight": "800",
                      }
                    }
                  >
                    Privacy Policy
                  </Txt>
                </Component>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Txt
                    color="light"
                    style={
                      Object {
                        "fontWeight": "800",
                      }
                    }
                  >
                    Privacy Policy
                  </Txt>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": "Privacy Policy",
                    "color": "light",
                    "style": Object {
                      "fontWeight": "800",
                    },
                  },
                  "ref": null,
                  "rendered": "Privacy Policy",
                  "type": [Function],
                },
                "type": [Function],
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": false,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
